lib/deliver/commands_generator.rb

```diff
@@ -49,7 +49,31 @@ def run
           Deliver::Runner.new(options).run
         end
       end
+      command :submit_build do |c|
+        c.syntax = 'deliver submit_build'
+        c.description = 'Submit a specific build-nr for review, use latest for the latest build'
+        c.action do |args, options|
+          options = FastlaneCore::Configuration.create(Deliver::Options.available_options, options.__hash__)
+          loaded = options.load_configuration_file("Deliverfile")
+          loaded = true if options[:description] || options[:ipa] || options[:pkg] # do we have *anything* here?
+
+          options[:skip_metadata] = true
+          options[:skip_screenshots] = true
+          options[:submit_for_review] = true
+          options[:force] = true
 
+          unless loaded
```

Comment:
hmmm, i am not sure - the action states `submit specific build for review` 
should we just set force to `false` and skip the others?